### PR TITLE
Add edit_url to feed

### LINKED ISSUE #438: Add edit_url to JSON
the edit_url parameter is part of the JSON spec now and adding that to WordPress JSON feeds will enable TSML UI (and perhaps Meeting Guide, someday) to send better feedback emails to admins. Example:

```
  "edit_url": "https://demo.code4recovery.org/wp-admin/post.php?post=5888&action=edit",
```

This is in service of [this request](https://github.com/code4recovery/tsml-ui/issues/127).

### PR DESCRIPTION
**Pull request by 97eighty**
Fixes #438

### CODE REVIEW

File: includes/functions.php
Comment (joshreisner) [inline includes/functions.php:1065]: apologies for the delay on reviewing this! i'm getting the following message when editing a meeting:

```
 Warning: Undefined variable $url in /Users/jreisner/Sites/valet/wordpress/wp-content/plugins/12-step-meeting-list/includes/functions.php on line 1065

Warning: Undefined variable $url in /Users/jreisner/Sites/valet/wordpress/wp-content/plugins/12-step-meeting-list/includes/functions.php on line 1065

Warning: Undefined variable $url in /Users/jreisner/Sites/valet/wordpress/wp-content/plugins/12-step-meeting-list/includes/functions.php on line 1065

Warning: Undefined variable $url in /Users/jreisner/Sites/valet/wordpress/wp-content/plugins/12-step-meeting-list/includes/functions.php on line 1065

Warning: Undefined variable $url in /Users/jreisner/Sites/valet/wordpress/wp-content/plugins/12-step-meeting-list/includes/functions.php on line 1065
```

if `tsml_feedback_url` is not set, what should it return? `null`?

File: includes/functions.php
Comment (joshreisner) [inline includes/functions.php:1121]: this looks good, except could we get it unescaped? i'm seeing `&nbsp;` instead of `&` in the URL: `"https://wordpress.test/wp-admin/post.php?post=213&amp;action=edit"`

**Comment by joshreisner** [comment]
@97eighty hope you dont mind, i pushed a commit that addresses my comments. should be good to go once we resolve the merge conflicts.

**Comment by brianw-area24** [comment]
I think this got fixed with another PR for TSML 3.13, therefore, I am closing this PR without merging it. If this is not the case, feel free to reopen it, and we'll have to work through the merge conflicts.

**Comment by joshreisner** [comment]
i don't think this code actually did get merged, i think there was some branch confusion around this time maybe? definitely not seeing this code in `master` right now. reopening

**Comment by joshreisner** [comment]
@brianw-area24 @97eighty fyi i've updated this PR to be just the single line about `edit_url` and would like to merge it with `develop`

i think what happened earlier was scope got added to add `feedback_url` and that got added in another PR, but `edit_url` slipped through

**Review by brianw-area24 — APPROVED** [review]
Looks good

**Pull request merged**
